iroha_client/src/client.rs

```diff
@@ -14,7 +18,7 @@ pub struct Client {
 
 /// Representation of `Iroha` client.
 impl Client {
-    pub fn new(configuration: &Configuration) -> Self {
+    pub fn new(configuration: &ToriiConfiguration) -> Self {
```

Comment:
Maybe make a type alias `ClientConfiguration` for `ToriiConfiguration` here?

Reply:
Better to create new configuration for clients only